string robot_name  # "Robo1"
string current_rack  # "R_A1"
bool task_complete  # true
